File: Old JAVA/application/TextConstant.java
package application;

public enum TextConstant {
	VERSION("0.1.30"),
	NOM("VetNutri"),
	STADE("Beta"),
	NBRATION("7")
	;
	


private String name = "";


//Constructeur
TextConstant(String name){
this.name = name;

}



public String nameToString(){
return name;
}

}

File: composeApp/src/commonMain/kotlin/fr/vetbrain/vetnutri_mp/Enumer/TextConstant.kt
package fr.vetbrain.vetnutri_mp.Enumer

enum class TextConstant(val value: String) {
    VERSION("3.3.07"),
    NOM("VetNutri"),
    STADE("Beta"),
    NBRATION("7");

    fun nameToString(): String = value
}
